finger/decorators.py

```diff
@@ -0,0 +1,29 @@
+"""
+A module to collect custom decorators that we use throughout fingerweb.
+"""
+
+from django.contrib.auth.decorators import login_required
+from django.http import HttpResponseForbidden
+from django.utils.functional import wraps
+
+
+def staff_or_superuser_required(view):
+    """
+    Require a logged in user that is both staff _and_ a superuser.
+
+    Usage:
+
+        from fingerweb.decorators import staff_or_superuser_required
+
+        @staff_or_organization_editor_required
```

Comment:
I guess this should be `@staff_or_superuser_required`?

Reply:
Lol, yes, obviously. I copy-typed from the wrong buffer.